### DISCUSSION TITLE
styled-components + Typescript

### DISCUSSION THREAD

**Discussion by samuelcelko** [jaredpalmer/razzle#1251] [Q&A] [answered]
Hey there, I'm getting the following console error in the browser after integrating styled-components
`index.js:1 Warning: Prop `className` did not match.`

I have installed `babel-plugin-styled-components` and put it in my babel config.
Has anyone come across this?

Thanks!

**Comment by imagine10255** [comment]
I have the same problem

**Comment by adrifer** [comment, marked as answer]
I had the same issue and I fixed it adding a .babelrc file and using this:
```
{
  "plugins": [
    [
      "styled-components",
      { "ssr": true, "displayName": true, "preprocess": false }
    ]
  ]
}
